src/commands/classes/Contest.command.ts

```diff
@@ -1,43 +1,169 @@
 import Command from "./Command";
-import { Message } from "discord.js";
+import { Message, Channel, TextChannel } from "discord.js";
 import drive from "googleapis";
-import fs from "fs";
+import fs, { watchFile } from "fs";
 import path from 'path';
 import validUrl from 'valid-url';
 import { dataflow } from "googleapis/build/src/apis/dataflow";
+import contestData from "../../../data/contestData.json";
```

Comment:
Import types should be grouped together. Curly brace imports should be grouped with curly brace imports. Also, order imports by alphabetical order in their respective groups.